zeebe/gateway-rest/src/test/java/io/camunda/zeebe/gateway/rest/controller/ProcessInstanceControllerTest.java

```diff
@@ -40,9 +43,10 @@ public class ProcessInstanceControllerTest extends RestControllerTest {
          "tenantId":"tenantId"
       }""";
   static final String PROCESS_INSTANCES_START_URL = "/v2/process-instances";
+  static final String CANCEL_PROCESS_URL = PROCESS_INSTANCES_START_URL + "/%s/cancellation";
 
   @Captor ArgumentCaptor<ProcessInstanceCreateRequest> requestCaptor;
```

Comment:
🔧 Let's rename this to `createRequestCaptor`, similarly to how you did this in the other cases.